### ISSUE TITLE
Feature Request: Comparing logged information across runs

### ISSUE THREAD

**Issue by s-j-99** [neptune-ai/neptune-optuna#53]
### Is your feature request related to a problem? Please describe.

Hi, I love how easy it is to compare images and certain run features across runs in the same project. However, my issue is that I track a final outcome of every Optuna study, by training a model with the best parameters in that run, and evaluating on an external dataset. Maybe I haven't figured out how, yet, but I would like to compare this metric across runs, since each run is using a different dataset. 

### Describe the solution you'd like

Essentially, I'm iterating over every dataset in a for-loop, initializing a new Neptune run and Optuna study, running hyperparameter optimization with a training and validation set. Once the optimization is done, I use the best parameters to train a new model, and test it on an external dataset. Then, I assign it using `run["external-test"]=test_score`. On Neptune's dashboard, I would like to compare this score across runs. How can I do this?

### Describe alternatives you've considered

I have considered just appending every test score to a list and then manually plotting it, which I guess is an extra step I could do. 

### Additional context
If more clarification is needed, I'm happy to provide. Thanks!

**Comment by SiddhantSadangi** [comment]
Hey @s-j-99 ,

Thank you for your patience!

Since the score is logged as a single value across runs, using Neptune's [side-by-side comparison tab](https://docs-legacy.neptune.ai/app/side-by-side/) would be the best option.

It would help you visualize changes in the `external-test` metric across the selected runs.

Is this what you are looking for?

**Comment by SiddhantSadangi** [comment]
Ah, yes!

In this case, you can use our [scatterplot widget](https://docs-legacy.neptune.ai/app/custom_dashboard/#scatter-plots) in custom dashboards.

![Image](https://github.com/user-attachments/assets/66c18952-1591-4788-8b4e-cf692cb1e654)

Since the x-axis only takes numbers and datetime, you can use the run's creation time as a proxy for the Run ID, and have the `external-test` metric as the y-axis.

Each point will then represent the `external-metric` for a run initialized at a particular `creation_time`. Hovering above the point shall show you the run ID as well.

Would this work for you?

**Comment by SiddhantSadangi** [comment]
Hey @s-j-99 ,
Been a while since I heard from you.

I am closing this issue now, but please feel free to reopen or reach out if you need any further support 🤗
